rootdir/Android.bp

```diff
@@ -0,0 +1,286 @@
+cc_prebuilt_binary {
+    name: "rdclean.sh",
+    srcs: ["vendor/bin/rdclean.sh"],
+    init_rc: ["vendor/etc/init/rdclean.rc"],
+    vendor: true,
+}
+
+prebuilt_etc {
+    name: "android.hardware.keymaster@4.0-service-qti",
```

Comment:
Note that the `.rc` postfix disappeared here. This was previously controlled by the `_STEM` in the Makefile.